builtins/dispatch.c

```diff
@@ -121,7 +121,10 @@ int32_t __get_system_isa() {
         _Bool avx512_amx_int8 =     (info2[3] & (1 << 25)) != 0;
         _Bool avx512_fp16 =         (info2[3] & (1 << 23)) != 0;
 #endif // !MACOS
-       // clang-format on
+        if (avx_vnni) {
```

Comment:
`dispatch.ll` is needed to be updated accordingly.

Reply:
Updated.